query/common.d.ts

```diff
@@ -269,6 +277,20 @@ export interface Bindings extends Iterable<[RDF.Variable, RDF.Term]> {
   ) => Bindings;
 }
 
+export interface Paths extends Iterable<[RDF.Variable, RDF.Term][]> {
```

Comment:
```suggestion
export interface Path extends Iterable<Bindings> {
```

I think you want a singular `Path` here. A QueryPaths will be a stream of Path objects. And also each item in the path can just use the `Bindings` type, right?